### PR TITLE
build: cap ipython version in tutorials for 3.8 compatibility

### PR DESCRIPTION
**Pull request by HLasse**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -72,6 +72,7 @@ tutorials = [
     "matplotlib",
     "datasets>=2.8.0",
     "scikit-learn>=1.1.1",
+    "ipython<=8.12",
```
Comment (KennethEnevoldsen) [inline pyproject.toml:75]: "ipython<=8.12; python_version < '3.9'"

Comment (KennethEnevoldsen) [reply pyproject.toml:75]: probably the way to do it

Comment (HLasse) [reply pyproject.toml:75]: what, didn't know you could do that. nice!

Comment (KennethEnevoldsen) [reply pyproject.toml:75]: I stole it from spacys requirements ;)

**Pull request merged**